tests/foreman/api/test_http_proxy.py

```diff
@@ -145,13 +145,13 @@ def test_positive_end_to_end(
 
 @pytest.mark.e2e
 @pytest.mark.upgrade
-@pytest.mark.rhel_ver_match('8')
+@pytest.mark.rhel_ver_match('9')
```

Comment:
recently I learned that there is an option to do `@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])` (default is currently 9), it could save you the effort to update this in the future, but up to you

Reply:
Yes I know about that default, but I haven't used here it since the client repo needs to be on par with the RHEL version, so change in default would break it anyway.